cmd/internal/client/delete.go

```diff
@@ -38,13 +38,17 @@ var CmdDeleteApp = &cobra.Command{
 		if len(args) != 0 {
 			return nil, cobra.ShellCompDirectiveNoFileComp
 		}
-		app, cleanup, _ := paas.NewCarrierClient(cmd.Flags())
+		app, cleanup, err := paas.NewCarrierClient(cmd.Flags())
 		defer func() {
 			if cleanup != nil {
 				cleanup()
 			}
 		}()
 
+		if err != nil {
+			return nil, cobra.ShellCompDirectiveNoFileComp
```

Comment:
when we have an error, we return no error?

Reply:
You are in the match code. There is no way to return an error

Reply:
You can only say 'no matches'